.github/workflows/ci.yml

```
@@ -26,35 +30,38 @@ jobs:
-        run: |
-          cd backend
-          python manage.py makemigrations --check
-      
-      - name: Run tests
-        run: |
-          cd backend
-          pytest --cov=. --cov-report=xml
+          pytest --splits 3 --group ${{ matrix.shard }} --cov=. --cov-report=xml
       
       - name: Upload coverage
         uses: codecov/codecov-action@v3
```

actionlint (1.7.12)

[error] the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)